distribution/docs/src/main/resources/_contents/running-contents.adoc

```diff
@@ -1384,6 +1384,8 @@ Monitor system logs with the *LogViewer*, a convenient "viewing portal" for inco
 
 The LogViewer displays the most recent 500 log messages by default, but will grow to a maximum of 5000 messages.
 
+The top left button labeled POLLING/PAUSED will toggle live retrieval of incoming logs from the backend.
+
 Log events can be filtered by:
 
 * Log level (`ERROR`, `WARNING`, etc).
```

Comment:
I know this isn't part of this change, but do you think we should clarify the log level filtering a little bit better, i.e., how the Karaf's logging level will affect the logs retrieved by the log viewer and how the log viewer filtering will only impact the logs being displayed?